src/Resources/Resources/help/Publish-AzBicepModule.md

```diff
@@ -0,0 +1,138 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.ResourceManager.dll-Help.xml
+Module Name: Az.Resources
+online version: https://docs.microsoft.com/powershell/module/az.resources/publish-azbicepmodule
+schema: 2.0.0
+---
+
+# Publish-AzBicepModule
+
+## SYNOPSIS
+Publishes a Bicep file to a registry.
+
+## SYNTAX
+
+```
+Publish-AzBicepModule -FilePath <String> -Target <String> [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Publish-AzBicepModule** cmdlet publishes a Bicep file to a registry as a module. To use the cmdlet, Bicep CLI (version 0.4.1008 or later) must be installed and added to PATH.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Publish-AzBicepModule -File main.bicep -Target br:{registry}/{moduleName}:{tag}
+```
+
+Publishes `main.bicep` to `br:{registry}/{moduleName}:{tag}`.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -FilePath
+Local path to the bicep file to publish.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -PassThru
+{{ Fill PassThru Description }}
```

Comment:
Indicates that this cmdlet returns a boolean result. By default, this cmdlet does not generate any output.